requirements.txt

```diff
@@ -24,12 +24,13 @@
 
 -e git+https://github.com/inveniosoftware/invenio-formatter.git#egg=invenio-formatter
 -e git+https://github.com/inveniosoftware/invenio-mail.git#egg=invenio-mail
--e git+https://github.com/inspirehep/invenio-query-parser.git@invenio3#egg=invenio-query-parser==0.4.2.dev20160310
+# FIXME temporary branch for testing
+-e git+https://github.com/inspirehep/invenio-query-parser.git@invenio3-inspire#egg=invenio-query-parser==0.6.0
 -e git+https://github.com/inspirehep/invenio-records.git@invenio3#egg=invenio-records
 -e git+https://github.com/inspirehep/invenio-pidstore.git#egg=invenio-pidstore
 
 -e git+https://github.com/inspirehep/workflow.git#egg=workflow==2.0.0.dev20160223
 -e git+https://github.com/inspirehep/invenio-workflows.git@invenio3#egg=invenio-workflows==1.0.0a1.dev20160126
--e git+https://github.com/jalavik/invenio-workflows-ui.git@master#egg=invenio-workflows-ui==0.1.0.dev20160000
+-e git+https://github.com/jmartinm/invenio-workflows-ui.git@test-invenio3-search#egg=invenio-workflows-ui==0.1.0.dev20160000
```

Comment:
Remember to update this remote branch and/or push it to inspirehep/invenio-workflows-ui repo